Formula/grpcurl.rb

```diff
@@ -0,0 +1,29 @@
+class Grpcurl < Formula
+  desc "Like cURL, but for gRPC"
+  homepage "https://github.com/fullstorydev/grpcurl"
+  url "https://github.com/fullstorydev/grpcurl/archive/v1.1.0.tar.gz"
+  sha256 "8c8af1643c4d549947a2f2198f5c21d634b2a24b31de975a5784841a9127c82f"
+
+  depends_on "go" => :build
+
+  def install
+    ENV["GOPATH"] = HOMEBREW_CACHE/"go_cache"
+    (buildpath/"src/github.com/fullstorydev/grpcurl").install buildpath.children
+    cd "src/github.com/fullstorydev/grpcurl/cmd" do
+      system "go", "build", "-ldflags", "-X main.version=#{version}",
```

Comment:
@crewmanmud, where does the value for `#{version}` come from? I don't see anything else in this file to indicate that it is v1.1.0. Does the `Formula` base class know how to parse it from the `url` field?

Reply:
Yes it does, it's pretty neat